.travis.yml

```diff
@@ -3,8 +3,8 @@ sudo: required
 services:
   - docker
 python:
-- '3.5'
 - '3.6'
+- '3.7'
```

Comment:
Didn't you say that you split this PR?

Reply:
I did. what you commented about is from PR #133.
I just rebased this branch over newly updated master, and somehow it references it here.

Reply:
This branch was intended to have 2 commits only:
 - 64842cf
 - 1f7f44f